Tests/DependencyInjection/Compiler/SerializerConfigurationPassTest.php

```diff
@@ -62,24 +62,24 @@ public function testShouldConfigureJMSSerializer()
     public function testShouldConfigureCoreSerializer()
     {
         $this->container->register('serializer', 'Symfony\Component\Serializer\Serializer');
-        $this->container->register('fos_rest.serializer.exception_wrapper_serialize_handler');
+        $this->container->register('fos_rest.serializer.exception_normalizer.jms');
 
         $compiler = new SerializerConfigurationPass();
         $compiler->process($this->container);
 
         $this->assertSame('fos_rest.serializer.symfony', (string) $this->container->getAlias('fos_rest.serializer'));
-        $this->assertTrue(!$this->container->has('fos_rest.serializer.exception_wrapper_serialize_handler'));
+        $this->assertFalse($this->container->has('fos_rest.serializer.exception_normalizer.jms'));
     }
 
-    public function testSerializerServiceSupersedesJmsSerializerService()
+    public function testJmsSerializerServiceSupersedesSerializerService()
     {
         $this->container->register('jms_serializer.serializer', 'JMS\Serializer\Serializer');
         $this->container->register('serializer', 'Symfony\Component\Serializer\Serializer');
 
         $compiler = new SerializerConfigurationPass();
         $compiler->process($this->container);
 
-        $this->assertSame('fos_rest.serializer.symfony', (string) $this->container->getAlias('fos_rest.serializer'));
+        $this->assertSame('fos_rest.serializer.jms', (string) $this->container->getAlias('fos_rest.serializer'));
```

Comment:
@xabbuh this was wrong according to the docs but it is technically a bc break. Is it fine for you anyway?